### DISCUSSION TITLE
I'm having trouble charging and USB-C voltage issues

### DISCUSSION THREAD

**Discussion by ErnestZCXu** [osy/XG_Mobile_Station#17] [Hardware] [answered]
Hello
I have implemented this project through JLCPCB
Currently available on ROG Ally
But I have the following problem:

1. ROG Ally is not charging after connecting to XG Mobile Station
2. After I connected the Hub to the USB-C interface of the XG Mobile Station, it could still operate at first (such as wireless keyboard and mouse), but then the Hub seemed to be burned...including the wireless receiver...
3. When running 3DMark, sometimes Windows displays white text on a blue background.

Could the above problem be caused by something incorrect on my part?
The equipment I use is as follows:

- Board: Please refer to the attachment
- Cable: https://en.accessoires-asus.com/laptop/cable/divers/asus-connection-cable-75-cm-62505-84262.htm
- Firmware: ST-LINK v2 (official version)
- Console: ROG Ally (2023)
- Power supply: Standard 500W ATX
- Graphics card: GTX780 (for testing, expected to be paired with RTX4080 Super)

PS: The above content is through Google Translate

[bom.xls](https://github.com/user-attachments/files/18218652/bom.xls)
![image](https://github.com/user-attachments/assets/9e6b4a7c-c02e-4d63-917e-04e65277ace6)
![image](https://github.com/user-attachments/assets/71529499-8c48-4ae3-ba25-686a3f97168b)

**Comment by osy** [comment, marked as answer]
Did you follow https://github.com/osy/XG_Mobile_Station/blob/main/Docs/Build_Guide.md ? You need impedance control:
![image](https://github.com/user-attachments/assets/70919a8b-58ec-4eb5-8e3c-1f67d2c7562d)

**Reply by ErnestZCXu** [reply]
Thanks for your reply
I actually made such a stupid question Orz...
Looks like I need to order the PCB again
What you originally ordered can be recycled directly, right?

**Reply by osy** [reply]
I don't know if it can be recycled.

**Reply by osy** [reply]
I don't remember what parts list I uploaded but I realized after testing that J14 (the 8 pin connector) is not needed and just the 24-pin is enough to power the board. That can save you a few dollars.

**Reply by ErnestZCXu** [reply]
Marvelous
So how do I remove J14?
Currently I only know that the 8 pin connector is an additional part that needs to be purchased.
Can I just ignore the J14 material shortage message and order the PCB?

**Reply by moni11811** [reply]
OSY I notice the impedance control set in your image does not match the one in the guide, would this need to be updated? what's the difference?

**Reply by osy** [reply]
@moni11811 sorry, the image is from a new revision that isn't released yet. Please follow the guide for accurate information.

**Comment by osy** [comment]
For USB:
![image](https://github.com/user-attachments/assets/b712c7ce-98fa-4ab7-8393-449daebc7978)

What hub are you using? Does it work connected directly to the Ally? Does your hub support PD passthrough?

**Reply by ErnestZCXu** [reply]
I use a set of USB-C to USB3*3 + RJ45 hub
And connect the wireless keyboard and mouse receiver
Works fine on Ally from the start
Later, it suddenly became inoperable and I found that the hub was very hot.
After the device cooled down, it stopped working when I tried to use it on another computer.

**Reply by osy** [reply]
What brand of hub? It is unlikely to be related to the board because the USB lanes are routed directly to the cable connector. It is just like an extension cable for USB. I think your USB hub may be faulty.

**Reply by ErnestZCXu** [reply]
TP-Link UE330
https://www.tp-link.com/en/home-networking/computer-accessory/ue330/

**Reply by osy** [reply]
That doesn't have PD passthrough. How are you powering the Ally?

**Reply by ErnestZCXu** [reply]
Ally power comes from USB-C?
I thought it was from an ATX power supply
However, I also tried connecting USB-C to the Ally original charger.
It was not charging at that time

**Reply by osy** [reply]
Like I wrote above, the dock does nothing with USB-C expect pass it through to the external port. It acts like an extension cable. I also pointed out that unlike standard USB-C cables, the orientation matters so you need to make sure the cable is not upside down. If that still didn't work for charging, you should take a picture of your setup because you might have not connected everything correctly.

**Reply by ErnestZCXu** [reply]
I understand
I guess I won't test it until I get the reordered board
I'm still worried that other USB Hubs will malfunction due to this
After all, the previously used TP-Link UE330 and keyboard and mouse receiver have been determined to be unusable.
